source/project-stories/reed-learning.erb

```diff
@@ -0,0 +1,156 @@
+  image: "/project-stories/reed-learning/learners-feedback.jpg"
+  image_description: "Reed Learning feedback from learners"
+
+team:
+  title:
+  people:
+    - full_name: "Leon Odey Knight"
+      image: "/project-stories/reed-learning/team/leon.png"
+      link: "/people/leon-odey-knight"
+
+    - full_name: "Tyler Savin"
+      image: "/project-stories/reed-learning/team/tyler.png"
+      link: "/people/tyler-savin"
+
+    - full_name: "Neil van Beinum"
+      image: "/project-stories/reed-learning/team/neil.png"
+      link: "/people/neil-van-beinum"
+
+    - full_name: "Henry Turner"
+      image: "/project-stories/reed-learning/team/henry.png"
+      link: "/people/henry-turner"
+
+    - full_name: "Martyn Evans"
+      image: "/project-stories/reed-learning/team/martyn.png"
+      link: "/people/martyn-evans"
+
+site:
+  link: ""
+  text: "Visit the site and check it out for yourself"
+
+links:
+  - title: "Lorem ipsum"
+    source: "Lorem ipsum"
+    type: "news"
+    link: ""
+
+  - title: "Lorem ipsum"
+    source: "Lorem ipsum"
+    type: "news"
+    link: ""
+
+related:
+  title: "Related"
+  articles:
+    - title: "Lorem ipsum"
+      type: "news"
+      link: ""
+
+    - title: "Lorem ipsum"
+      type: "blog"
+      link: ""
+
+    - title: "Lorem ipsum"
+      type: "project story"
+      link: ""
+
+
+---
+
+<%= partial 'project-stories/page_header' %>
+
+<div class="container" data-header-waypoint>
+  <%= partial 'project-stories/full_image', locals: { section: current_page.data.header } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.introduction } %>
+
+  <section class="highlight--reed-learning">
+    <div class="highlight__tile--two">
+      <p class="highlight__large-text"><%= current_page.data.statistics_1.number %><p>
+      <p class="highlight__small-text"><%=current_page.data.statistics_1.text %></p>
+      <p class="highlight__large-text"><%= current_page.data.statistics_2.number %><p>
+      <p class="highlight__small-text"><%=current_page.data.statistics_2.text %></p>
+    </div>
+  </section>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.innovating_product } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.cultural_change_quote } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.understanding_customer } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.getting_confidence_quote } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.new_customers } %>
+
+  <%= partial 'project-stories/quote', locals: { section: current_page.data.team_learning_quote } %>
+
+  <%= partial 'project-stories/text_tile', locals: { section: current_page.data.innovation_strategy } %>
+
+  <%= partial 'project-stories/project_team' %>
```

Comment:
oh I'm so fed up with using gimp (program) already - especially that the caricatures are so different, i.e. Leon one has only a bust and very long hands so it has completely different proportions than other caricatures after the cropping. I'll try to fix it, but if it'll take too much time I'm delegating that to the designer :)
Thanks for the tips!